dataverse-webapp/src/main/webapp/editdatafiles.xhtml

```diff
@@ -61,7 +58,7 @@
                         <div  class="button-block">
                             <p:outputPanel id="filesButtons" >
                                 <div jsf:rendered="#{!(empty EditDatafilesPage.fileMetadatas and (datasetPage || EditDatafilesPage.showFileUploadFragment()))}">
-                                    <p:commandButton tabindex="1000" id="save" value="#{bundle.saveChanges}" onclick="checkNewlyRestricted();PF('blockFileForm').show();" />
+                                    <p:commandButton tabindex="1000" id="save" value="#{bundle.saveChanges}" onclick="clickDatasetSaveButton();PF('blockFileForm').show();" />
```

Comment:
Hmm would it not be enough to just remove this button and use "datasetSave" button?

Reply:
Yes, it will be enough